lib/internal/vm/module.js

```diff
@@ -252,8 +246,46 @@ function validateInteger(prop, propName) {
   }
 }
 
+const getWrapFromModule = async (m, scriptOrModule, linker) => {
```

Comment:
Any reason why these are not function declarations? Ditto below.

Reply:
not particularly, should i change them to function declarations?

Reply:
I believe that to be the prevalent style.